src/main/java/me/totalfreedom/totalfreedommod/command/Command_wipeflatlands.java

```diff
@@ -24,7 +25,21 @@ public boolean run(final CommandSender sender, Player playerSender, Command cmd,
             player.kickPlayer("Server is going offline for flatlands wipe, come back in a few minutes.");
         }
 
-        server.shutdown();
+        if (!plugin.cpb.isEnabled())
```

Comment:
Would probably suggest adding a config value and then checking before getting to this as to if the server has Core Protect bridge enabled. If so you wouldnt need to do this check as you should do it on server startup.